plugins/legacy-plugin-chart-pivot-table/test/PivotTable.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { getTimeFormatterForGranularity } from '@superset-ui/core';
```

Comment:
This file probably doesn't need to be `tsx`, `ts` should be enough. Let's also add the license headers to all new files (check for instance `controlPanel.ts`)